users/models.py

```diff
@@ -68,6 +68,8 @@ class User(AbstractUser):
     username: None = None
     email = models.EmailField(unique=True)
     last_activity = models.DateTimeField(auto_now_add=True, db_index=True)
+    create_post: bool = models.BooleanField(default=True)
```

Comment:
What does this field do? Please describe using help text.

Reply:
And ifi this describes if used can create post, should be `can_create_post`

Reply:
Okey